django_ipam/base/models.py

```diff
@@ -28,8 +30,9 @@ def __str__(self):
 
     def clean(self):
         for subnet in swapper.load_model("django_ipam", "Subnet").objects.filter().values():
-            if ip_network(self.subnet).overlaps(subnet["subnet"]):
-                raise ValidationError({'subnet': _('Subnet overlaps with %s') % (subnet["subnet"])})
+            if self.id != subnet["id"]:
```

Comment:
I find this weird.

A better alternative would be:

```
if self.id == subnet["id"]:
    continue
if ip_network(self.subnet).overlaps(subnet["subnet"]):
    pass
```

or also a single `if` statement